lib/rightnow/objects/incident.rb

```diff
@@ -231,9 +231,9 @@ def update_incident
 
   def find_incident
     Nokogiri::XML::Builder.new(encoding: "UTF-8") do |xml|
-      xml.QueryObjects("xmlns" => "urn:messages.ws.rightnow.com/v1_2") do
-        xml.Query("SELECT Incident FROM Incident i WHERE i.ID = '#{id}'")
-        xml.ObjectTemplates("xmlns:object" => "urn:objects.ws.rightnow.com/v1_2", "xmlns:xsi" => "http://www.w3.org/2001/XMLSchema-instance", "xsi:type" => "object:Incident") do
+      xml.QueryObjects("xmlns" => "urn:messages.ws.rightnow.com/v1_4") do
+        xml.Query("SELECT Incident FROM Incident i WHERE i.ID = #{id}")
```

Comment:
This is actually the only functional change. Should have been caught the last time we tested the upgrade